Installing-Oppia-using-Docker.md

```diff
@@ -103,6 +103,8 @@ For systems with 8GB RAM:
 
    ![Docker Setup Terminal Logs](images/dockerSetupTerminalLogs.png)
 
+3. **Verify the development server is up**: Open your browser and navigate to `http://localhost:8181/`. You should see the Oppia development server running.
```

Comment:
Rephrase "Verify the development server is up" to something along the lines of Access oppia server

Reply:
@gp201 okay

Reply:
@gp201 I would like to add a image to the repo

Reply:
@gp201 I could not find a home page image in the /images directory

Reply:
add the image to this directory. https://github.com/oppia/oppia-web-developer-docs/tree/develop/images/install

Reply:
@gp201 I have made the necessary changes can you please check